package.json

```diff
@@ -69,7 +69,7 @@
     "@testing-library/react": "14.0.0",
     "@types/htmlparser2": "^3.10.3",
     "@types/lodash": "^4.14.161",
-    "@types/node": "^16.0.0",
+    "@types/node": "^20.0.0",
```

Comment:
Er dette problematisk for feks edlib som er på node 17?

Reply:
Dette er en dev dependency, så nei. Med mindre edlib bruker vercel-deployen vår burde dette ikke påvirke de